src/table/table.stories.ts

```diff
@@ -71,6 +72,26 @@ const getProps = (more = {}) => {
 	}, more);
 };
 
+function getModelWithDisabledRows() {
+	const disabledModel = new TableModel();
+	const row1 = new TableRow();
+	row1.disabled = true;
+	row1.push(new TableItem({data: "Name 1"}));
```

Comment:
Since you're extending the `Array` constructor, I believe you can pass the items in when you create the row 

```typescript
const row1 = new TableRow(
	row1.push(new TableItem({data: "Name 1"}));
	row1.push(new TableItem({data: "Disabled 1"}));
);
row1.disabled = true;
```